[2023-03-01 13:12:33,979] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-12-11T17:05:00+00:00 [queued]>
[2023-03-01 13:12:34,192] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-12-11T17:05:00+00:00 [queued]>
[2023-03-01 13:12:34,193] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:12:34,194] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 13:12:34,195] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:12:34,302] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-12-11T17:05:00+00:00
[2023-03-01 13:12:34,312] {standard_task_runner.py:52} INFO - Started process 21349 to run task
[2023-03-01 13:12:34,359] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-12-11T17:05:00+00:00', '--job-id', '5124', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmp9s2eib_b', '--error-file', '/tmp/tmpa3wv19ai']
[2023-03-01 13:12:34,386] {standard_task_runner.py:77} INFO - Job 5124: Subtask get_name
[2023-03-01 13:12:34,910] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-12-11T17:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 13:12:35,721] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T17:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-11T17:05:00+00:00
[2023-03-01 13:12:36,270] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 13:12:36,469] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221211T170500, start_date=20230301T131234, end_date=20230301T131236
[2023-03-01 13:12:37,332] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-01 13:12:37,490] {local_task_job.py:146} INFO - Task exited with return code 0
